Samples/PowerShell/Groups.ps1

```diff
@@ -26,6 +26,28 @@ $swis = Connect-Swis -host $hostname -cred $cred
 # Creating a new group with initial Cisco and Windows devices.
 #
 
+<# Filter samples w/ all operators:
+
+IP_Address is 192.168.10.10
+filter:\Orion.Nodes[IP_Address='192.168.10.10')]
```

Comment:
It looks like the slash is backwards on this one - `filter:\`.

Reply:
There were a couple others with the same problem, but aside from that this looks great! Thanks!

Reply:
AH, whoops, good catch - thank you!

Reply:
@mrxinu - Got that updated. Thanks!